### DISCUSSION TITLE
skywalking agent not support request header had sw8-correlation

### DISCUSSION THREAD

**Discussion by JihanLin** [apache/skywalking#10705] [General]
frist,i am sorry,my english is poor.
i am using cyborg-flow,but not using APISIX for gateway.i think this framework is just add sw8-correlation to request header.
so i put sw8-correlation to my API request header ,but when i debug skywalking agent 's TomcatInvokeInterceptor class, then  ContextManager.createEntrySpan ,carrier.isValid() is false, it reject load sw8-correlation,it must need sw8 in header,but i don't need traceId.
how i do?thanks.

**Comment by wu-sheng** [comment]
`cyborg-flow` is owned by SphereEX. Please ask them for all you need.

**Comment by lihuagang03** [comment]
We use HTTP header "sw8-correlation" to pass custom data. But it didn't take effect.

```
sw8-correlation | c3c4X3VzZXJJZA==:MTIzNDU2
```

<img width="974" alt="image" src="https://github.com/apache/skywalking/assets/1811851/3692fac7-8daf-4e9a-a152-fe96f68f5365">
